es.go

```diff
@@ -1174,6 +1174,34 @@ func (c *Client) SnapshotAllIndices(repository string, snapshot string) error {
 	return err
 }
 
```

Comment:
@sarwaan001 I'd like to see a docstring added but otherwise looks good.

Reply:
added docstring